examples/print.rs

```diff
@@ -1,6 +1,7 @@
-use nalgebra as na;
-
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
I think that this example works on wasm, doesn't it?

Reply:
Good catch! Reverted changes on examples/print.rs and examples/urdf.rs.